sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -309,6 +309,10 @@ protected ServiceBusProcessor()
         /// or Subscription.
         /// Implementation is mandatory.
         /// </summary>
+        /// <remarks>
+        /// It is not recommended that the state of the processor be managed directly from within this handler; requesting to start or stop the processor may result in
```

Comment:
Is it worth making the distinction that the deadlock would only happen when this is awaited?

Reply:
I considered it... what caught me was the thought that we really don't want to encourage using it in the handlers.    We want to guide towards other patterns.